beard/similarity/pairs.py

```diff
@@ -273,3 +274,40 @@ def transform(self, X):
         X2 = X[:, n_features:]
 
         return np.abs(X1 - X2)
+
+class SetJaccard(TransformerMixin):
```

Comment:
Can you also rename the class to `JaccardSimilarity`? For consistency with the other transformers defined in the same file.